index.html

```diff
@@ -22,13 +22,22 @@
     <title>Gci16.fossasia.org by fossasia</title>
 </head>
 <body>
-    <header>
+    <header style="padding-bottom:10px;">
         <div class="inner">
             <h1>Gci16.fossasia.org</h1>
             <h2>GCI Website 2016</h2>
             <a href="//github.com/fossasia/gci16.fossasia.org" class="button">
                 <small>View project on</small> GitHub</a>
         </div>
+        <div class="sec1">
+            <a class="sec2" href="/#mentors">Mentors</a>
+            <a class="sec2" href="/#contributors">Contributers</a>
+            <a class="sec2" href="/#student-sec">Students</a>
+            <a class="sec2" href="/#blog">Blog</a>
+            <a class="sec2" href="/#mbdyn">Logos</a>
+            <a class="sec2" href="/#issues">Issues</a>
+            <a class="sec2" href="/#soc-med">Social</a>
+        </div>
```

Comment:
please remove the slashes preceding the `#` in the links, that is why the links are not working on your fork.